docs/zoneddatetime.md

```diff
@@ -1411,7 +1428,7 @@ Usage example:
 zdt = Temporal.ZonedDateTime.from('1995-12-07T03:24:30.000003500+01:00[Europe/Paris]').withCalendar('japanese');
 
 // Year in japanese calendar is year 7 of Heisei era
-zdt.year; // => 7
+zdt.eraYear; // => 7
```

Comment:
This example doesn't make so much sense anymore with `year` in the Japanese calendar already being the signed year (and only provisionally determined by the polyfill)
Maybe choose a different example, like use the Hebrew calendar and show the difference between `zdt.month` and `zdt.getISOFields().isoMonth`?